kite-redis/Dockerfile

```diff
@@ -1 +1,6 @@
-FROM redis:5.0.3-alpine
\ No newline at end of file
+FROM redis:5.0.3-alpine
+
+COPY ./kite-redis.conf /etc/kite-redis.conf
+
+EXPOSE 6379
```

Comment:
We shouldn't need to expose port 6369 as it is already part if the redis:5.0.3-alpine image